src/index.js

```diff
@@ -0,0 +1,36 @@
+import './style.css';
+import displayTasks from './populateTaskList.js';
+import AddToDoList from './addRemoveTask.js';
+
+const taskList = new AddToDoList();
+displayTasks(taskList);
+
+const addTodoBtn = document.getElementById('addTask');
+addTodoBtn.addEventListener('click', () => {
```

Comment:
- It would be better if you don't add extra unneeded words to the variable names.
- Better put 'addButton'